### PR TITLE
feat: add example `13_path-alias`

### PR DESCRIPTION
**Pull request by himself65**
Fixes: https://github.com/dai-shi/waku/issues/109

### CODE REVIEW

**Comment by dai-shi** [comment]
@himself65 Can you retry this?

**Comment by himself65** [comment]
Ok

File: examples/13_path-alias/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "waku-example",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dev": "waku dev --with-ssr",
+    "build": "waku build",
```
Comment (dai-shi) [inline examples/13_path-alias/package.json:8]: Is it intentional to not have `--with-ssr`?

File: examples/13_path-alias/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { defineConfig } from 'vite';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+export default defineConfig({
+  plugins: [tsconfigPaths()],
```
Comment (dai-shi) [inline examples/13_path-alias/vite.config.ts:5]: I see. This does the job.

File: examples/13_path-alias/package.json (hunk 1 above)
Comment (dai-shi) [reply examples/13_path-alias/package.json:8]: Let's add it and see how it goes.
```suggestion
    "build": "waku build --with-ssr",
```

**Pull request merged**